src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.WebKey/Microsoft.Azure.KeyVault.WebKey.csproj

```diff
@@ -4,8 +4,8 @@
     <PackageId>Microsoft.Azure.KeyVault.WebKey</PackageId>
     <Description>Microsoft Azure Key Vault WebKey Class Library</Description>
     <AssemblyTitle>Microsoft Azure Key Vault WebKey</AssemblyTitle>
+    <VersionPrefix>2.0.8-alpha2</VersionPrefix>
```

Comment:
any reason you want to tag as alpha. Usually in nuget we tag such releases are preview?